### PR TITLE
use package logrus to log ,  support custom format

### PR DESCRIPTION
**Pull request by MouceL**

### CODE REVIEW

File: slime-framework/controllers/destinationrule_controller.go [hunk 1]
```diff
@@ -35,7 +33,7 @@ import (
 // DestinationRuleReconciler reconciles a DestinationRule object
 type DestinationRuleReconciler struct {
 	client.Client
-	Log    logr.Logger
+	Log    *logrus.Entry
```
Comment (YonkaFang) [inline slime-framework/controllers/destinationrule_controller.go:36]: do we need this field while having migrating to this new log lib?

File: slime-framework/controllers/destinationrule_controller.go [hunk 2]
```diff
@@ -44,17 +42,17 @@ type DestinationRuleReconciler struct {
 
 func (r *DestinationRuleReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
 	_ = context.Background()
-	_ = r.Log.WithValues("destinationrule", req.NamespacedName)
+	log := r.Log.WithField("destinationRule", req.NamespacedName)
```
Comment (YonkaFang) [inline slime-framework/controllers/destinationrule_controller.go:45]: global var?

File: slime-modules/lazyload/controllers/process.go [hunk 3]
```diff
@@ -160,13 +160,13 @@ func (r *ServicefenceReconciler) ReconcileNamespace(req ctrl.Request) (ret ctrl.
 	// refresh service fenced status
 	services := &corev1.ServiceList{}
 	if err = r.Client.List(ctx, services, client.InNamespace(req.Name)); err != nil {
-		r.Log.Error(err, "list services failed", "ns", req.Name)
+		r.Log.Errorf("list services %s failed, %+v", req.Name,err)
```
Comment (YonkaFang) [inline slime-modules/lazyload/controllers/process.go:163]: need to gofumpt all changed files

File: slime-modules/lazyload/controllers/servicefence_controller.go [hunk 4]
```diff
@@ -136,29 +137,29 @@ func (r *ServicefenceReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error
 func (r *ServicefenceReconciler) refreshSidecar(instance *lazyloadv1alpha1.ServiceFence) error {
 	sidecar, err := newSidecar(instance, r.env)
 	if err != nil {
-		r.Log.Error(err, "servicefence生产sidecar的过程中发生错误")
+		r.Log.Errorf("servicefence生产sidecar的过程中发生错误,%+v",err)
```
Comment (YonkaFang) [inline slime-modules/lazyload/controllers/servicefence_controller.go:140]: change to en

File: slime-modules/limiter/controllers/multicluster/controller.go [hunk 5]
```diff
@@ -153,27 +149,25 @@ func (c *Controller) addMemberCluster(secretName string, s *corev1.Secret) {
 		// clusterID must be unique even across multiple secrets
 		if _, ok := c.cs[clusterID]; !ok {
 			if len(kubeConfig) == 0 {
-				log.Info("Data '%s' in the secret %s in namespace %s is empty, and disregarded ",
+				logrus.Infof("Data '%s' in the secret %s in namespace %s is empty, and disregarded ",
```
Comment (YonkaFang) [inline slime-modules/limiter/controllers/multicluster/controller.go:152]: import as log, hide log impl

**Review by YonkaFang — APPROVED** [review]

**Comment by YonkaFang** [comment]
resolves #74

**Comment by MouceL** [comment]
resolves #74

**Pull request merged**